identify/extensions.py

```diff
@@ -183,11 +183,12 @@
     'vue': {'text', 'vue'},
     'war': {'binary', 'zip', 'jar'},
     'wav': {'binary', 'audio', 'wav'},
-    'wkt': {'text', 'wkt'},
     'whl': {'binary', 'wheel', 'zip'},
+    'wkt': {'text', 'wkt'},
     'woff': {'binary', 'woff'},
     'woff2': {'binary', 'woff2'},
     'wsgi': {'text', 'wsgi', 'python'},
+    'xhtml': {'text', 'xml', 'xhtml'},
```

Comment:
Should `xhtml` have `html` type as well?

Reply:
yeah probably, sorry missed this first time around!

Reply:
Sorry, saw it just now!

Reply:
Shall I add it still in another PR?

Reply:
up to you!